scanpy/preprocessing/__init__.py

```diff
@@ -21,6 +21,7 @@
     subsample,
 )
 
+
```

Comment:
Could you remove this and other blank space diffs?  This makes the git blame hard to follow for future users